nixos/maintainers/scripts/azure/create-azure.sh

```diff
@@ -1,6 +1,6 @@
-#! /bin/sh -e
+#! /bin/sh -ex
```

Comment:
maybe add a `u`?

Reply:
Oh this change was accidental, lol. But I’ll add a u anyway (and remove the x).